package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yote",
-  "version": "0.8.0",
+  "version": "0.8.0.1",
```

Comment:
also note that i dont think this needs to be or necessarily should be in step with the yote versions. this program is separate and has functionality outside of fugitivelabs/yote, and may separate bugs and fixes.

Reply:
also needs to be incremented every time we deploy it